pkg/nodediscovery/nodediscovery.go

```diff
@@ -284,6 +292,17 @@ func (n *NodeDiscovery) StartDiscovery() {
 	}
 }
 
+func (n *NodeDiscovery) UpdateLocalNode(fillLocalNode bool) {
+	n.localNodeLock.Lock()
+	defer n.localNodeLock.Unlock()
+
+	if fillLocalNode {
+		n.fillLocalNode()
+	}
+
+	n.updateLocalNode()
+}
```

Comment:
I don't see any reference to this function in the commit description? Wrong commit maybe?

Reply:
Forgot to update the commit message